Sources/FCM/Helpers/FCM+JWT.swift

```diff
@@ -4,16 +4,15 @@ import JWT
 
 extension FCM {
     func generateJWT() throws -> String {
-        gAuthPayload.updateIfNeeded()
```

Comment:
This line shouldn't be removed, it should be replaced with `gAuthPayload.update()` instead, because it cause problems with token refreshing #6